website/src/components/Tasks/CreateTask.tsx

```diff
@@ -55,16 +57,37 @@ export const CreateTask = ({
                 {t(getTypeSafei18nKey(`tasks:${taskType.id}.instruction`))}
               </Text>
             )}
-            <TrackedTextarea
-              text={inputText}
-              onTextChange={textChangeHandler}
-              thresholds={{ low: 20, medium: 40, goal: 50 }}
-              textareaProps={{
-                placeholder: t(getTypeSafei18nKey(`tasks:${taskType.id}.response_placeholder`)),
-                isDisabled,
-                isReadOnly: !isEditable,
-              }}
-            />
+            {!isEditable ? (
+              <Suspense fallback={inputText}>
+                <RenderedMarkdown markdown={inputText}></RenderedMarkdown>
+              </Suspense>
```

Comment:
since these 3 lines are identical to the one below, can we hoist them to a `useMemo`?